Tests/FunctionalTests/TestRig/Sources/FTRScrollViewController.xib

```diff
@@ -19,66 +22,66 @@
         </placeholder>
         <placeholder placeholderIdentifier="IBFirstResponder" id="-2" customClass="UIResponder"/>
         <view contentMode="scaleToFill" id="1">
-            <rect key="frame" x="0.0" y="0.0" width="600" height="600"/>
+            <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
```

Comment:
May want to add another scroll view container instead of modifying existing ones because the existing ones are already used in tests.